spec/re_captcha/configuration_spec.rb

```diff
@@ -26,20 +26,86 @@
     end
 
     describe '#valid?' do
+      let(:errors) { double('Array', :<< => nil) }
+
+      it 'resets errors' do
+        expect(instance).to receive(:reset_errors)
```

Comment:
avoid expectations on private methods.

What you really want to test is that if you call `.valid?` two times for the same object errors will not be duplicated